browser/src/control/jsdialog/Widget.Edit.ts

```diff
@@ -115,6 +115,8 @@ class EditWidget {
 				this.builder,
 			);
 		}
+		if (e.key == 'Enter' && this.data.activateDefault)
```

Comment:
if custom callback is used (above `if (typeof callbackToUse === 'function')` - we don't want to do any additional actions. Parent container has full control.

I think it should be moved to the `else` branch only.